<?xml version="1.0"?>

<launch>

        <arg name="testName"   default="aria_test"/>

		<param name="use_sim_time" value="true"/>
        <node pkg="rosbag" type="play" name="data" output="screen" args="/home/aeroscout/aria_mapping.bag --pause --clock "/>

        <node pkg="lidar_eskf" type="bag_to_pcd" name="bag_to_pcd" output="screen" required="true">
            <param name="record_time"                               value="10"/>
    		<param name="max_range"                                 value="90"/> 
    		<param name="min_range"	                                value="0.5"/>
    		<param name="robot_frame"                               value="/base_frame"/>
            <param name="bt_file_name"                              value="$(find lidar_eskf)/map/$(arg testName).bt"/>
            <param name="octomap_resolution"                        value="0.1"/> 
    		<param name="save_pcd_true"                             value="true"/>
	    	<param name="save_bt_true"                              value="false"/>
	    	<param name="imu_distort"                               value="true"/>
	    	<param name="pcd_file_name"                             value="$(find lidar_eskf)/map/$(arg testName).pcd"/>

	    	<remap from="/imu"                                      to="/microstrain/imu"/>
	    	<remap from="/cloud"                                    to="/velodyne_points"/>
	    	<remap from="/scan"                                     to="/scan"/>
	</node>

	<!-- <node pkg="rosbag" type="record" name="mapping_bag" output="screen" args="-O $(find lidar_eskf)/map/$(arg testName).bag /velodyne_points /microstrain/imu /tf" />-->
	
 </launch>
